### PR TITLE
[eslint] linting fixed after adding lint rules

### PR DESCRIPTION
**Pull request by abu-hasib**

### CODE REVIEW

**Comment by abu-hasib** [comment]
@MimaoQiuQiu these are fixes for the lint error when i added lining. Please check and approve.

**Comment by mihrab34** [comment]
@abu-hasib  great work. I did notice that you have other changes apart from the eslint and pre-husky commit.

File: cubeseed_login/components/create-invoice.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+import Image from "next/image"
+import { useState } from "react"
+import billto from "@cs/public/bill-to.png"
+import payto from "@cs/public/pay-to.png"
+import Input from "./pieces/input"
+import Textarea from "./pieces/textarea"
+import Table from "./pieces/table"
+import CreateSucess from "./create-success"
+import { BillType, OrderType, SummaryType } from "@cs/types"
+import { DashboardHeading } from "./header"
```
Comment (mihrab34) [inline cubeseed_login/components/create-invoice.tsx:12]: Some of these changes could have been made in another PR or write clear commit messages for them so it's easily trackable.

**Review by mihrab34 — CHANGES_REQUESTED** [review]
can you re-arrange some of the components you added to the `cubeseed_login/components` folder and move it into the` src/components`
Having them in different places makes it confusing to review

**Review by mihrab34 — COMMENTED** [review]
We can just open a new issue for the folder restructure

**Review by mihrab34 — APPROVED** [review]

**Comment by abu-hasib** [comment]
Not really, the app uses router app

**Pull request merged**